### PR TITLE
frontend: Catalog Config Patches

### PR DESCRIPTION
**Pull request by jdslaugh**
### Description
- ~Adds fullWidth functionality to Error/Alert components~
- Re-Exports the ToggleButton from @material-ui/lab
- Adds textTransform functionality to typography
- Moves linkGroup check inside of QuickLinks component
- Keeps navigate params for redirects

### Testing Performed
manual

### CODE REVIEW

**Review by smonero — APPROVED** [review]

File: frontend/packages/core/src/Feedback/alert.tsx [hunk 1]
```diff
@@ -76,10 +77,17 @@ const iconMappings = {
 export interface AlertProps
   extends Pick<MuiAlertProps, "severity" | "action" | "onClose" | "elevation" | "variant"> {
   title?: React.ReactNode;
+  fullWidth?: boolean;
```
Comment (dschaller) [inline frontend/packages/core/src/Feedback/alert.tsx:80]: hmmm I'm curious about this. When looking at the [storybook component](https://storybook.clutch.sh/?path=/story/core-feedback-alert--with-title&args=variant:standard) its rendering as full width of the available space so we might not need this.

File: frontend/workflows/projectCatalog/src/config/index.tsx [hunk 2]
```diff
@@ -80,20 +91,24 @@ const Config: React.FC<ProjectDetailsConfigWorkflowProps> = ({ children, default
...
+            description="Edit your projects' settings. Some changes may require additional approval from another owner of the project."
           />
         </Grid>
         {configPages && configPages.length > 1 ? (
           <Grid item xs={12}>
             <Tabs value={selectedPage} centered>
               {configPages.map((page, i) => (
-                <Tab label={page.props.title} onClick={() => setSelectedPage(i)} />
+                <Tab
+                  key={page.props.title}
```
Comment (dschaller) [inline frontend/workflows/projectCatalog/src/config/index.tsx:108]: 🎉

File: frontend/packages/core/src/Feedback/alert.tsx (hunk 1 above)
Comment (jdslaugh) [reply frontend/packages/core/src/Feedback/alert.tsx:80]: For whatever reason I couldn't get it to go full width. The alert was fine, but the error rendering the alert was breaking it and I couldn't get it to override. I'll take another pass though.

Comment (jdslaugh) [reply frontend/packages/core/src/Feedback/alert.tsx:79]: Bah figured it out, was an issue with the grid. I've removed the properties now so should be good :)

Comment (dschaller) [reply frontend/packages/core/src/Feedback/alert.tsx]: 👍

**Review by dschaller — APPROVED** [review]

**Pull request merged**